locales/cy/company-inactive.json

```diff
@@ -0,0 +1,7 @@
+{
+    "companyInactiveTitle" : "Company is not active Welsh",
+    "canNotRegister" : "You cannot register ",
```

Comment:
Instead of concatenating in njk can we parameterise it like this?  "You cannot register {0} as an authorised agent because it is not an active company."